Gemfile

```diff
@@ -3,6 +3,10 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in youtube-dl.rb.gemspec
 gemspec
 
+group :test do
+  gem 'activesupport', '< 5.0'
```

Comment:
this could be `gem 'activesupport', '< 5.0' unless RUBY_VERSION >= '2.2.2'` to continue testing against latest `activesupport` in later ruby versions.